src/st/class-wpml-pb-loader.php

```diff
@@ -64,12 +68,24 @@ public function __construct(
 
 	}
 
-	private static function load_hooks() {
+	/**
+	 * @param bool $has_page_builder_strategy
+	 */
+	private static function load_hooks( $has_page_builder_strategy ) {
 		$hooks = [
-			WPML_PB_Handle_Post_Body::class,
 			WPML\PB\Compatibility\Toolset\Layouts\HooksFactory::class,
```

Comment:
This class for Toolset will be remove in another task (it has been migrated there now).